A66_National_13_09_01.LICENSE.md

```diff
@@ -0,0 +1,3 @@
+Licensed under a Creative Commons Attribution 4.0 Licence. 
```

Comment:
I don't think these license files are needed. A reference to the publishing authoritys website should be enough. As far as I can tell it is this one https://github.com/icsm-au/transformation_grids and a clear statement in `GRIDS.md` which license is attached to each grid.